### PR TITLE
Clean up some style issues in the documentation

### PR DESCRIPTION
**Pull request by serapio**
1. use `:doc:` and `:ref:` links instead of generic web links
2. use sphinx style links for web links too
3. bold some things that were meant to be bold

### CODE REVIEW

File: source/versions/history.rst [hunk 1]
```diff
@@ -7,55 +7,55 @@ Package History
...
 
 Bug Fixes and Other Changes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Fixed an [issue](https://github.com/cisco/mindmeld/pull/400) with the build of docker containers.
-- Set [default tokenizer](https://github.com/cisco/mindmeld/pull/397) for all languages to Whitespace.
-- Refactor how the resource loader is [initialized in NativeQA](https://github.com/cisco/mindmeld/pull/391)
-- Fix a [bug](https://github.com/cisco/mindmeld/pull/390) related to updating the MM version.
-- Remove a redundant warning in [TextPreparationPipeline creation](https://github.com/cisco/mindmeld/pull/385).
+- Fixed an `issue <https://github.com/cisco/mindmeld/pull/400>`__ with the build of docker containers.
```
Comment (berlinguist) [inline source/versions/history.rst:30]: Definitely not a big issue, and it looks like things render perfectly fine either way, but there are some differences between using a single underscore _ and two of them __ in some of these links in this file

**Review by berlinguist — APPROVED** [review]
lgtm, just a small comment about links

File: source/versions/history.rst (hunk 1 above)
Comment (serapio) [reply source/versions/history.rst:30]: The double underscore just tells doc compiler not to treat these as named links, to squash some warnings.

**Review by vrdn-23 — APPROVED** [review]
lgtm

**Pull request merged**